homeassistant/components/tuya/camera.py

```diff
@@ -81,6 +89,23 @@ async def stream_source(self) -> str | None:
             "rtsp",
         )
 
+    async def async_create_stream(self) -> Stream | None:
+        """Create a Stream for stream_source."""
+        if not self.stream:
+            return await super().async_create_stream()
+
+        # Tuya streams urls are one time urls. Updating source of existing stream.
+        if not self._create_stream_lock:
```

Comment:
This private member variable in the parent class`_create_stream_lock` is not part of the entity API, and changing it from the child class is not something we want as it would be non-obvious.

Reply:
I can rename it to _update_stream_lock.